fix: use relative path in file backup

netchampfaris opened the pull request:
Because of #22142, get_site_path returns an absolute path. It broke file restores on sites.

<img width="878" alt="image" src="https://github.com/frappe/frappe/assets/9355208/0fb6acce-ba65-45b7-bfe9-59ce99ca18bd">

ankush commented:
@netchampfaris we can also probably revert that PR and jusy resolve path instead of making them abspath.

netchampfaris committed:
fix: resolve path instead of absolute path

#22142 made the change to absolute path but it affected file backups.
Resolve path should probably be enough in most cases.

ankush commented:
It seems there's no test at all for file restore :smile: 

https://github.com/frappe/frappe/blob/develop/frappe/tests/test_commands.py

Also, why aren't we compressing by default. https://github.com/frappe/press/issues/1052

netchampfaris committed:
chore: unskip restore test

netchampfaris committed:
chore: run restore test only on mariadb
